test/Window/Context.test.cpp

```diff
@@ -21,28 +21,24 @@ TEST_CASE("[Window] sf::Context", runDisplayTests())
     {
         STATIC_CHECK(!std::is_copy_constructible_v<sf::Context>);
         STATIC_CHECK(!std::is_copy_assignable_v<sf::Context>);
-        STATIC_CHECK(!std::is_nothrow_move_constructible_v<sf::Context>);
-        STATIC_CHECK(!std::is_nothrow_move_assignable_v<sf::Context>);
+        STATIC_CHECK(std::is_nothrow_move_constructible_v<sf::Context>);
+        STATIC_CHECK(std::is_nothrow_move_assignable_v<sf::Context>);
     }
 
     SECTION("Construction")
     {
         const sf::Context context;
-
         CHECK(context.getSettings().majorVersion > 0);
     }
 
     SECTION("Version String")
     {
         sf::Context context;
-
         CHECK(context.setActive(true));
 
         using glGetStringFuncType = const char*(GLAPI*)(unsigned int);
-
-        auto glGetStringFunc = reinterpret_cast<glGetStringFuncType>(sf::Context::getFunction("glGetString"));
-
-        REQUIRE(glGetStringFunc);
+        auto glGetStringFunc      = reinterpret_cast<glGetStringFuncType>(sf::Context::getFunction("glGetString"));
+        REQUIRE(!!glGetStringFunc);
```

Comment:
This converts the function pointer to a boolean which fixes an issue that leads to UBSan failing under certain conditions. I've discussed this with the Catch2 maintainer and it appears to be an issue with my Clang 17 installation. 

There are actually other places in our tests that trigger this same error. I'm unsure if I bother fixing this here if I don't also fix it elsewhere.